### PR TITLE
app-dicts/myspell-es: version bump to 2.3, update HOMEPAGE and SRC_URI

### PR DESCRIPTION
**Pull request by BorisCarvajal**
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: app-dicts/myspell-es/myspell-es-2.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+DESCRIPTION="Spanish dictionaries for myspell/hunspell"
+HOMEPAGE="https://github.com/sbosio/rla-es"
+LICENSE="GPL-3 LGPL-3 MPL-1.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+IUSE=""
+
+src_prepare() {
+	# remove license files
+	rm -rf {GPL,LGPL,MPL}*
```
Comment (Moiman) [inline app-dicts/myspell-es/myspell-es-2.3.ebuild]: remove -f from rm and add missing die.

File: app-dicts/myspell-es/myspell-es-2.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MUTATIONS="es_ANY es_AR es_BO es_CL es_CO es_CR es_CU es_DO es_EC es_ES es_GT
```
Comment (mgorny) [inline app-dicts/myspell-es/myspell-es-2.3.ebuild]: Please use a bash array here as well.

File: app-dicts/myspell-es/myspell-es-2.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+SRC_URI=""
+for i in ${MUTATIONS}; do
+	MYSPELL_DICT+=(
+		"${i}.dic"
+		"${i}.aff"
+	)
+	SRC_URI+=" https://github.com/sbosio/rla-es/releases/download/v${PV}/${i}.oxt -> ${i}-${PV}.oxt"
+done
+unset i
```
Comment (mgorny) [inline app-dicts/myspell-es/myspell-es-2.3.ebuild]: I suppose you can unset `MUTATIONS` as well, while at it.

**Comment by mgorny** [comment]
The package looks good. However, it seems that you either don't have a Gentoo Bugzilla account or used the wrong email address in metadata.xml. We're using this to assign bugs, so please make sure it corresponds to Bugzilla account.

**Comment by BorisCarvajal** [comment]
The account has been created just now.

**Comment by mgorny** [comment]
Thanks a lot! Confirmed and merging now.

**Pull request closed without merging**